python/pyspark/sql/tests.py

```diff
@@ -238,7 +238,6 @@ def test_basic_functions(self):
         df = self.sqlCtx.jsonRDD(rdd)
         df.count()
         df.collect()
-        df.schema
```

Comment:
I suppose that this was testing whether `df.schema` threw an error or not.  Can you re-enable this statement and try adding a comment to exclude this from Pylint to confirm that the exclusion comment works?